File: library/src/main/java/cn/wandersnail/http/callback/ProgressListener.java
package cn.wandersnail.http.callback;

/**
 * 下载进度监听
 * <p>
 * date: 2019/8/23 15:21
 * author: zengfansheng
 */
public interface ProgressListener {
    /**
     * 进度更新
     *
     * @param progress 已完成的大小
     * @param max      总大小
     */
    void onProgress(long progress, long max);
}

File: library/src/main/java/com/snail/network/download/ProgressListener.kt
package com.snail.network.download

/**
 * 时间: 2017/7/8 20:23
 * 作者: zengfansheng
 * 邮箱: 43068145@qq.com
 * 功能: 下载进度监听
 */

internal interface ProgressListener {
    /**
     * 下载进度
     * @param readLength    已下载长度
     * @param contentLength 总长度
     */
    fun onProgress(readLength: Long, contentLength: Long)
}
